student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative './person'
+
+class Student < Person
+  def initialize(classroom, age, name: 'Unknown', parent_permission: true)
+    super(age, name, parent_permission)
```

Comment:
- [ ] I tried creating a student and got an error relating to the arguments expected. Since your `name` and `parent_permission` attributes are defined using the colon notation, you need to also assign your arguments in the `#super` method, like so:

  ```
  super(age, name: name, parent_permission: parent_permission)
  ```